Sources/NIOHTTP2/HTTP2PipelineHelpers.swift

```diff
@@ -96,11 +96,12 @@ extension Channel {
     public func configureHTTP2Pipeline(mode: NIOHTTP2Handler.ParserMode,
                                        initialLocalSettings: [HTTP2Setting] = nioDefaultSettings,
                                        position: ChannelPipeline.Position = .last,
+                                       targetWindowSize: Int32 = 65535,
```

Comment:
Unfortunately, for `public func`s we can't add new arguments to the methods, even when they're defaulted, as it is still API breaking. Instead, we need to add a new function that has the same signature as the previous one _plus_ the extra argument, not defaulted.

Do you mind doing that?